getEvent/index.ts

```diff
@@ -0,0 +1,32 @@
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import getConnection from '../shared/services/db';
+import { getEventSchema } from '../shared/event/event.schema';
+import FunctionError from '../shared/services/error';
+import EventModel from '../shared/event/event.model';
+
+const getResObject = (statusCode: Number | undefined, responseMessage: any) => {
+    return { status: statusCode ?? process.env.SERVER_ERROR_CODE, body: responseMessage };
```

Comment:
There is no reason to save status codes in the env file, it won't change. 
I write this here but you should change it in the other files as well.

```suggestion
    return { status: statusCode ?? 500, body: responseMessage };
``